terminus-ui/logo/src/logo.component.md

```diff
@@ -42,26 +43,22 @@ Solid logos are white by default, but can also be black (Terminus Dark) or gray
 <ts-logo type="mark-solid" logoColor="black"></ts-logo>
 ```
 
-Search for `TS_LOGO_COLORS` to see all allowed logo colors.
-
+See for `TS_LOGO_COLORS` to see all allowed logo colors.
 
 ## Sizing
-The four main logos are full-size so they will adapt to the *width* of their container. Changing the height of the parent container will result in weird spacing issues, and is not recommended.
-
 
+The four main logos are full-size so they will adapt to the width of their container.
 
 ## Special Cases
-Any logo with a gradient will not honor a logoColor.
-Sometimes marketing will request a logo for marketing purposes and those logos have special rules. 
-
 
+- Any logo with a gradient will not honor a logoColor.
 
-## Available
+## Available logos
 
-| Name                | Description                                                                                                 |
-|--------------------:|:------------------------------------------------------------------------------------------------------------|
-| `full-account-hub`  | Special logo for Account Hub, includes spacing and color, should be used according to marketing standards   |
-| `full-gradient`     | Default. Gradient mark with logo gray "terminus" text                                                       |
-| `full-solid`        | Mark with "terminus" text, white by default, accepts logoColor                                              |
-| `mark-gradient`     | T formed by arrow on top and blocks below to form a T in negative space, postive space has gradient         |
-| `mark-solid`        | T formed by arrow on top and blocks below to form a T in negative space, postive space is white by default  |
+|               Name | Description                                                                                                 |
+|-------------------:|:------------------------------------------------------------------------------------------------------------|
+| `full-account-hub` | Special logo for Account Hub, includes spacing and color, should be used according to marketing standards   |
+|    `full-gradient` | Default. Gradient mark with logo gray "terminus" text                                                       |
+|       `full-solid` | Mark with "terminus" text, white by default, accepts logoColor                                              |
+|    `mark-gradient` | T formed by arrow on top and blocks below to form a T in negative space, positive space has gradient        |
+|       `mark-solid` | T formed by arrow on top and blocks below to form a T in negative space, positive space is white by default |
```

Comment:
Do you try to align them up to the right?

Reply:
Not always (I don't really have a pattern there..) But this was done by someone else and I didn't change it - my webstorm markdown plugin just auto-formats markdown tables so it 'corrected' the original placement but the rendered table stayed the same.